modules/backend/assets/js/october.sidenav-tree.js

```diff
@@ -206,7 +206,7 @@
             $('ul li', $li).each(function(){
                 var $itemLi = $(this)
 
-                $itemLi.toggleClass('hidden', $.inArray(this, visibleItems) == -1)
+                $itemLi.toggleClass('hidden', visibleItems.indexOf(this) == -1)
```

Comment:
@DanHarrin could you change it to strict comparison here `===`. 

Please note that if you have a `var` (might be applicable to jQuery object as well),  you should always check if it's not a `null or undefined` (could be simplified to `visibleItems && visibleItems.indexOf(this) === -1)`. Otherwise, you'd get an error `'indexOf' function is not defined`.

Reply:
All done @w20k 👍